src/win32/wazuh-installer.wxs

```diff
@@ -177,10 +177,17 @@
                         <File Id="LOCAL_INTERNAL_OPTIONS.CONF" Name="local_internal_options.conf" Source="default-local_internal_options.conf" KeyPath="yes" />
                     </Component>
 
-                    <Component Id="OSSEC.CONF" DiskId="1" Guid="26C3265E-EFC8-488D-8D19-397A0C44C071" NeverOverwrite="yes" Permanent="yes">
+                    <Component Id="OSSEC_PRE6.CONF" DiskId="1" Guid="26C3265E-EFC8-488D-8D19-397A0C44C071" NeverOverwrite="yes" Permanent="yes">

```

Comment:
Change the GUIDs, use `26C3265E-EFC8-488D-8D19-397A0C44C071` for the default `ossec.conf` and `71947f0e-56d6-4e14-8539-214dff6ca2be` for the `ossec-pre6.conf`, I think is better to keep the same GUID for the ossec.conf in systems post-vista.